app/src/main/res/values/strings.xml

```diff
@@ -927,4 +927,11 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="quest_pedestrian_crossing_island">"Does this pedestrian crossing have an island?"</string>
     <string name="quest_surface_detailed_answer_impossible_title">Describe surface</string>
     <string name="open_url">Open URL</string>
+    <string name="quest_tactile_paving_kerb_title">Is there tactile paving on this kerb for this crossing?</string>
+    <string name="quest_kerb_height_title">What is the height of this kerb?</string>
+    <string name="quest_kerb_height_flush">Same level as road surface</string>
+    <string name="quest_kerb_height_lowered">Lowered, still above road surface</string>
+    <string name="quest_kerb_height_raised">High kerb</string>
+    <string name="quest_kerb_height_answer_no_kerb">There is no kerb on this side of crossing</string>
+    <string name="quest_kerb_height_lowered_ramp">Curb cut</string>
```

Comment:
Shouldn't this be British English like the ones above?

```suggestion
    <string name="quest_kerb_height_lowered_ramp">Kerb cut</string>
```

Reply:
Good point, though SC is in US English with UK English handled as translation for such cases. I will fix this.